cypress/e2e/newUser.cy.js

```diff
@@ -0,0 +1,68 @@
+describe('Should display new user page with Nav and input fields', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/#/new-user')
+    })
+
+    it('Should display a navBar', () => {
+        cy.get('header').should('be.visible')
+          .get('header').should('contain', 'WeatherTogether')
+          .get('header').should('contain', 'Login')
+          .get('header').should('contain', 'New User')
+    })
+
+    it('Should should display newUser fixture', () => {
+        cy.get('.newuser-form').should('be.visible')
+          .get('.newuser-form').should('contain', 'E-mail:')
+          .get('.newuser-form').should('contain', 'Username: ')
+          .get('.newuser-form').should('contain', 'Password:')
+          .get('.newuser-form').should('contain', 'Confirm Password')
+          .get('.newuser-form').should('contain', 'Create Account')
+        cy.get('img').should('be.visible')
+          .get('img').should('have.attr', 'src', '/weather1-fe/static/media/logo_480.67f44ac75dde7b7a1a89.png')
+        cy.get('.email').should('contain', '')
+        cy.get('.username').should('contain', '')
+        cy.get('.password').should('contain', '')
+        cy.get('.confirm-password').should('contain', '')
+    })
+})
+
+describe('Should allow New User creation with checks', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/#/new-user')
+    })
+
+    it('Should allow User to input into fields', () => {
+        cy.mockUserInput()
+    })
+
+    it('Should allow for User creation and route to Login', () => {
+        cy.mockUserInput()
```

Comment:
since this is pretty specific to testing user signup I'm not sure pulling this out into a command is helping much. we want to extract commands for logic that's repeated over and over.

Reply:
Makes sense. Done.